docs/developer-guide.md

```diff
@@ -60,6 +61,21 @@ You can update necessary files as follows:
 make generate
 ```
 
+## Controller Flags
+
+Below is a list of command-line flags accepted by Katib controller:
+
+| Name                            | Type        | Default             | Description                                                                                                             |
+| ------------------------------- | ----------- | ------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| cert-localfs                    | bool        | false               | Store the webhook cert in local file system                                                                             |
+| enable-grpc-probe-in-suggestion | bool        | true                | Enable grpc probe in suggestions                                                                                        |
+| experiment-suggestion-name      | string      | "default"           | The implementation of suggestion interface in experiment controller                                                     |
+| metrics-addr                    | string      | ":8080"             | The address the metric endpoint binds to                                                                                |
+| trial-resources                 | value       | null                | The list of resources that can be used as trial template, in the form: Kind.version.group (e.g. TFJob.v1.kubeflow.org)  |
```

Comment:
I think we can define the type here, WDYT @annajung @gaocegege 
```suggestion
| trial-resources                 | []schema.GroupVersionKind       | null                | The list of resources that can be used as trial template, in the form: Kind.version.group (e.g. TFJob.v1.kubeflow.org)  |
```

Reply:
That sounds good to me! Updated with the suggestion